tests/functional/api/test_import_export.py

```diff
@@ -64,3 +66,15 @@ def test_project_import_export(gl, project, temp_dir):
         count += 1
         if count == 15:
             raise Exception("Project import taking too much time")
+
+
+def test_project_remote_import(gl):
+    with pytest.raises(gitlab.exceptions.GitlabHttpError) as err_info:
+        gl.projects.remote_import(
+            "ftp://whatever.com/url", "remote-project", "remote-project", "root"
+        )
+    assert err_info.value.response_code == 400
+    assert (
+        "File url is blocked: Only allowed schemes are https"
+        in err_info.value.error_message
+    )
```

Comment:
We could also use a `match=` argument here, but we can do this in a follow-up cleanup along with the list issue.

Reply:
I considered this, but I wasn't sure of checking the status code through a regex, bcoz apparently match takes a string literal or regex. So went with this. We can discuss this on the PR for S3 support.